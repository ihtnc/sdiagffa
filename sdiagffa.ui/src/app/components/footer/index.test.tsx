import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import Footer from '.'

describe('Footer component', () => {
  it('should match snapshot', () => {
    const component = render(<Footer />);
    expect(component).toMatchSnapshot();
  })
});