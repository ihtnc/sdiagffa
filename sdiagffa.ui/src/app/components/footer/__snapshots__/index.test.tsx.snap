// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="
      text-xs
      opacity-60
      text-swblack dark:text-swyellow
      text-center
    "
      >
        "Six degrees in a galaxy far, far away" by 
        <a
          class="underline"
          href="https://github.com/ihtnc"
          target="_blank"
        >
          @ihtnc
        </a>
        .
        <br />
        Based on the concept, 
        <a
          class="underline"
          href="https://en.wikipedia.org/wiki/Six_Degrees_of_Kevin_Bacon"
          target="_blank"
        >
          six degrees of Kevin Bacon
        </a>
        .
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="
      text-xs
      opacity-60
      text-swblack dark:text-swyellow
      text-center
    "
    >
      "Six degrees in a galaxy far, far away" by 
      <a
        class="underline"
        href="https://github.com/ihtnc"
        target="_blank"
      >
        @ihtnc
      </a>
      .
      <br />
      Based on the concept, 
      <a
        class="underline"
        href="https://en.wikipedia.org/wiki/Six_Degrees_of_Kevin_Bacon"
        target="_blank"
      >
        six degrees of Kevin Bacon
      </a>
      .
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
